### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by direnc** [directus/directus#7222] [Archive]
I plan to post encrypted data to Directus API but need to decrypt it before saving. What is the easiest way to accomplish this?

**Comment by rijkvanzanten** [comment]
How are you going to encrypt the data before sending it to the Directus API? Is this server-to-server communication? 

The easiest way would probably be with a custom endpoint: https://docs.directus.io/guides/api-endpoints/

**Comment by direnc** [comment]
Yes, it is server to server, I’m going to encrypt the data with PHP before sending.

**Reply by rijkvanzanten** [reply]
Yeah then you'll definitely have to setup a custom endpoint, as the way you decrypt the data will have to be written custom in order to be compatible with your specific encryption method in PHP 👍🏻
